### DISCUSSION TITLE
issue: input state isTouched is always false

### DISCUSSION THREAD

**Discussion by mzumberi** [react-hook-form/react-hook-form#10352] [Q&A]
### Version Number

^7.43.9

### Codesandbox/Expo snack

https://codesandbox.io/s/awesome-thunder-myjokb?file=/src/slider.tsx

### Steps to reproduce

Hi, 
i'm facing a field state (isTouched) issue related on a input type range being always false

see codeSandbox [link](https://codesandbox.io/s/awesome-thunder-myjokb?file=/src/slider.tsx)

 

### Expected behaviour

whenever i change the slider value i expect isTouched to be true

### What browsers are you seeing the problem on?

Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
If onBlur event is not getting fired, then touched would be registered, please double check your component API.

**Comment by mzumberi** [comment]
hi, thanks for your reply
i've just followed the code linked into the get started docs https://codesandbox.io/s/react-hook-form-adapting-existing-form-ts-uzfxm

any suggestion to fix the behaviour?
